src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,114 @@
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { NavLink, useParams } from 'react-router-dom';
+import React from 'react';
+import classNames from 'classnames';
+import { LoaderEnum } from '../types/loader';
+
+interface PeopleTable {
```

Comment:
```suggestion
interface PeopleTableProps {
```
<img width="631" alt="image" src="https://github.com/user-attachments/assets/822b6957-8362-4336-b80f-01ae1efe92cb" />